### PR TITLE
feat: Notify users of plan changes

### PR DESCRIPTION
**Pull request by taratatach**
When we receive a plan change from the cloudery (i.e. a new feature
set that's not a sponsorship), we send a notification (preferably a
push one) to the user to invite them to reload their mobile app and/or
their browser page so flags can be refreshed.

This is especially useful when the user is expecting to gain new
features.

### CODE REVIEW

File: assets/mails/notifications_planchanged.mjml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{{define "content"}}
```
Comment (nono) [inline assets/mails/notifications_planchanged.mjml:1]: [nit] I would have put an underscore in the filename between `plan` and `changed`

**Review by nono — APPROVED** [review]

File: assets/mails/notifications_planchanged.mjml (hunk 1 above)
Comment (taratatach) [reply assets/mails/notifications_planchanged.mjml:1]: I decided to copy the existing pattern for notification e-mails templates but I agree with you.
Let's change that.

File: assets/mails/notifications_plan_changed.mjml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{{define "content"}}
```
Comment (nono) [inline assets/mails/notifications_plan_changed.mjml:1]: Can you also update https://github.com/cozy/cozy-stack/blob/master/docs/assets.md#emails ?

Comment (taratatach) [reply assets/mails/notifications_plan_changed.mjml:1]: Yes I saw that it was missing so I added it.

**Pull request merged**